### DISCUSSION TITLE
DP training with low GPU usage

### DISCUSSION THREAD

**Discussion by tonystarkiss** [deepmodeling/deepmd-kit#1039] [Q&A]
![image](https://user-images.githubusercontent.com/30335530/130913422-4dde2ac9-0e0a-494f-8b92-5c6a3e7fa99c.png)

I trained my model with dp, but I find my GPU SM usage only 50~60%, usually the deep learning training the GPU usage can up to 100%, WHY?

**Comment by njzjz** [comment]
You can consider increasing the batch size.

**Reply by tonystarkiss** [reply]
the batch_size is "auto", I do not know when I is set with "auto" how to caculate the batch size? when I set batch size=4, the GPU usage up to 90%.

**Reply by njzjz** [reply]
Please read the [document](https://docs.deepmodeling.org/projects/deepmd/en/latest/train/train-input.html#training-training-data-batch-size). You are free to set an atomic number other than 32.
